File: crud-sharp-mongodb/crudsharp-mongodb/libCrud/Person.cs
using MongoDB.Bson;
using MongoDB.Bson.Serialization.Attributes;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace libCrud
{
    public class Person
    {
        private ObjectId id;
        public ObjectId Id
        {
            get { return id; }
            set { id = value; }
        }

        private string name;
        public string Name
        {
            get { return name; }
            set { name = value; }
        }

        private string email;
        public string Email
        {
            get { return email; }
            set { email = value; }
        }

        private PhysicalPerson physicalPerson = new PhysicalPerson();
        public PhysicalPerson PhysicalPerson
        {
            get { return physicalPerson; }
            set { physicalPerson = value; }
        }

        public override string ToString()
        {
            return String.Format("{0} - {1} - {2} - {3} - {4} - {5}", this.Id, this.Name, this.Email, this.PhysicalPerson.Salary, this.PhysicalPerson.Birthday, this.PhysicalPerson.Gender);
        }
    }
}

File: java-mongo-app/JavaMongoApp/src/javamongoapp/entities/Person.java
package javamongoapp.entities;

import org.bson.types.ObjectId;

public class Person {

    private ObjectId id;
    private String name;
    private String email;
    private PhysicalPerson physicalPerson = new PhysicalPerson();

    public ObjectId getId() {
        return id;
    }

    public void setId(ObjectId id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public PhysicalPerson getPhysicalPerson() {
        return physicalPerson;
    }

    public void setPhysicalPerson(PhysicalPerson physicalPerson) {
        this.physicalPerson = physicalPerson;
    }

    @Override
    public String toString() {
        StringBuilder stb = new StringBuilder();
        stb.append("ID: ");
        stb.append(this.getId());
        stb.append(" |Name: ");
        stb.append(this.getName());
        stb.append(" |E-mail: ");
        stb.append(this.getEmail());
        stb.append(" |Salary: ");
        stb.append(this.getPhysicalPerson().getSalary());
        stb.append(" |Birthday: ");
        stb.append(this.getPhysicalPerson().getBirthday());
        stb.append(" |Gender: ");
        stb.append(this.getPhysicalPerson().getGender());
        return stb.toString();
    }
}
